packages/media/src/formatMsToHMS.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import formatDuration from './formatDuration';
+
+/**
+ * Converts length in milliseconds to HH:MM:SS.SSS format.
+ *
+ * @param {number} ms Original length in milliseconds.
+ * @param {boolean} isForDisplay If the output is for displaying to the user.
+ * @return {string} Formatted length.
+ */
+function formatMsToHMS(ms, isForDisplay = false) {
```

Comment:
@spacedmonkey @swissspidy I've added a `isForDisplay` flag here now for formatting ms to H:M:S format either for displaying for the user or not. Let me know if you think it's better to split it out to two separate utils as Jonny had [in his PR](https://github.com/google/web-stories-wp/pull/9143/files#diff-6e2ce5102e38dfd909c13da6ea6cdd896c0a7b5e8cce6a094606a7f73c54cdf8R28).

Reply:
Separate functions seem cleaner, boolean flag params are often considered an anti-pattern.

The shared part (getting an array of hours, minutes, seconds from ms) could be extracted into a shared function

Reply:
Yep, I also felt like it got a bit confusing. Will split it up.